javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -553,10 +553,8 @@ public static boolean hasInheritedMeta(AnnotationMirror anno) {
             case TYPE_USE:
                 return EnumSet.noneOf(ElementKind.class);
             default:
-                // TODO: Add actual case to check for the enum constant and return Set containing
-                // ElementKind.MODULE.  (Java 11)
-                if (elementType.name().contentEquals("MODULE")) {
-                    return EnumSet.noneOf(ElementKind.class);
+                if (elementType.name().equals("MODULE")) {
+                    return EnumSet.of(ElementKind.valueOf("MODULE"));
```

Comment:
This is now inconsistent with the Javadoc.  Could you please just delete the following lines in the Javadoc:
```
* <p>If the Element is MODULE, the empty set is returned. This is so that this method can
* compile with Java 8.
```

Reply:
Oops, was just getting to this now.  Thanks for handling it.